clients/ui/frontend/src/app/pages/modelCatalog/components/ModelCatalogStringFilter.tsx

```diff
@@ -9,14 +9,16 @@ import { useCatalogStringFilterState } from '~/app/pages/modelCatalog/utils/mode
 
 const MAX_VISIBLE_FILTERS = 5;
 
-type ModelCatalogStringFilterProps<K extends ModelCatalogStringFilterKey> = {
+type ArrayFilterKey = Exclude<ModelCatalogStringFilterKey, ModelCatalogStringFilterKey.USE_CASE>;
```

Comment:
Interesting, I'm curious why this is necessary - in theory we could render a ModelCatalogStringFilter using checkbox options for multiple use_case values